grpc-gateway/service/service.go

```diff
@@ -47,13 +47,13 @@ func New(ctx context.Context, config Config, fileWatcher *fsnotify.Watcher, logg
 
 	pool, err := ants.NewPool(config.Clients.Eventbus.GoPoolSize)
 	if err != nil {
-		server.Close()
+		_ = server.Close()
```

Comment:
This could be problematic if the implementation ever changes and returns a real error. I'm 99% sure that the implementer won't think about needing to update all calls of `server.Close` to check error.

Reply:
fixed

Reply:
There are `s.Close()` and `grpcServer.Close()` calls that ignore the error. :innocent:

Reply:
Look good. ;)